web/src/main/kotlin/no/nav/su/se/bakover/web/routes/grunnlag/FormuevilkårJson.kt

```diff
@@ -23,14 +27,18 @@ internal data class VurderingsperiodeFormueJson(
     val periode: PeriodeJson,
 )
 
-internal fun Vilkår.Formue.toJson(): FormuevilkårJson {
+internal fun Vilkår.Formue.toJson(satsFactory: SatsFactory): FormuevilkårJson {
+
     return FormuevilkårJson(
         vurderinger = when (this) {
             is Vilkår.Formue.IkkeVurdert -> emptyList()
             is Vilkår.Formue.Vurdert -> vurderingsperioder.map { it.toJson() }
         },
         resultat = resultat.toFormueStatusString(),
-        formuegrenser = this.formuegrenser.toJson(),
+        // TODO jah + jacob:  Denne bør flyttes til et eget endepunkt i de tilfellene vi ikke har fylt ut formuegrunnlaget/vilkåret enda.
```

Comment:
TODO tenke på flyt, behov for vurdering av formue for hver enkelt periode med forskjellige formuegrenser etc. Hvordan vise i frontend, hvordan validere at under grense etc frontend.